rust/private/utils.bzl

```diff
@@ -179,6 +179,23 @@ def expand_locations(ctx, env, data):
     """
     return dict([(k, _expand_location(ctx, v, data)) for (k, v) in env.items()])
 
+def expand_arg_locations(ctx, args, data):
```

Comment:
I don't know if this over-complicates the implementation but I think it'd be nicer to add an optional `args` filed to `expand_location`? Maybe make `env` optional as well but assert that one *or* the other is set? Could be talked out of this pattern though.

Reply:
That seems more complicated than what I have now, and would probably require more documentation too. I'd prefer to leave it as is, unless others want to refactor it.

Reply:
If that's the case then what do you think about renaming `expand_location` to `expand_env_locations`? For some reason the `expand_location` + `expand_arg_locations` really irks me 😅

Reply:
Done.